### PR TITLE
Add upgrade CLI command

### PR DESCRIPTION
**Pull request by Santoshziffity**
## Purpose of this pull request

This pull request (PR)  for Add upgrade CLI command 

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.3/config-guide/deployment/pipeline/example/shared-configuration.html
-  https://devdocs.magento.com/guides/v2.3/config-guide/deployment/pipeline/example/environment-variables.html
-  https://devdocs.magento.com/guides/v2.3/config-guide/deployment/pipeline/example/cli.html

### CODE REVIEW

File: src/_includes/config/split-deploy/example_build-sync.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ To update your build system:
    git pull mconfig m2.2_deploy
    ```
 
+1. Run the upgrade command:
+
+   ```bash
+   php bin/magento setup:upgrade
```
Comment (dobooth) [inline src/_includes/config/split-deploy/example_build-sync.md:16]: We don't do `php` at the start of commands anymore, correct?

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/_includes/config/split-deploy/example_build-sync.md (hunk 1 above)
Comment (dobooth) [reply src/_includes/config/split-deploy/example_build-sync.md:16]: Ok....so you can remove the `php`.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Review by hguthrie — APPROVED** [review]

**Comment by dobooth** [comment]
Hey @Santoshziffity Just want to make sure: This is a big command to run...and it's in an include file. Are we sure this is appropriate for all instances where the include is used? Does any backup need to happen before running it?

**Comment by ajithkumar-maragathavel** [comment]
> Hey @Santoshziffity Just want to make sure: This is a big command to run...and it's in an include file. Are we sure this is appropriate for all instances where the include is used? Does any backup need to happen before running it?

@dobooth  Santoshziffity will not be available for quite some time. we might need to check this with someone else.

File: src/_includes/config/split-deploy/example_build-sync.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ To update your build system:
    git pull mconfig m2.2_deploy
    ```
 
+1. Run the upgrade command:
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply src/_includes/config/split-deploy/example_build-sync.md]: If possible, please clearly state what is going on here. I used Upgrade Magento as a suggestion, but that's maybe not the most apt description. I have seen `Update the database schema and data.` used before. Please confirm what this does in this particular step.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Running this command at this stage requires a lot more information and explanation to be added as firstly it REQUIRES not only Magento to be installed on the system that the command is run on but also access to a Magento database. 

Even though the intention is to allow code files to be committed to the repository.

It also depends on the CI/CD workflow used.

EG if this were being run on a local machine, then Magento would need to be fully working - what happens if the database is not available?

I feel more work needs to be done to cover all eventualities.

**Comment by dobooth** [comment]
I think I would more comfortable closing this PR for now. Once @Santoshziffity is back in action, we can discuss the optimal solution for this issue. Thanks all for the conversation. This is what makes open source a good idea.

**Pull request closed without merging**